string scene_name
---
string all_points